src/tests/dictionaryConcepts/container/CreateConcept.test.jsx

```diff
@@ -253,4 +253,43 @@ describe('Test suite for dictionary concepts components', () => {
       expect(wrapper.find('CreateConceptForm').props().mappings[0].map_type).toEqual(inputValue);
     });
   });
+
+  it('should follow the provided path when the Leave button is clicked', () => {
+    const mockPath = localStorage.getItem('dictionaryPathName');
+    const instance = wrapper.find('CreateConcept').instance();
+    const spy = jest.spyOn(instance, 'selectConfirm');
+    wrapper.find('CreateConcept').setState({ show: true }, () => {
+      wrapper.find('button#generalConfirmButton').simulate('click');
+      expect(spy).toHaveBeenCalled();
+      expect(props.history.push).toHaveBeenCalledWith(mockPath);
+    });
+  });
+
+  it('should hide the modal when the Stay button is clicked', () => {
+    const instance = wrapper.find('CreateConcept').instance();
```

Comment:
What is 'CreateConcept'?

Reply:
`CreateConcept ` is part of the parent component which is defined within `wrapper` in the `beforeEach` section of the Test suite. Lines `62-67`

Reply:
What exact is it?

Reply:
`CreateConcept` is a component and this line is extracting an instance of that component. I'll rename the variable name to `createComponentInstance`.

Reply:
I have renamed it as promised.